pkg/event/controller.go

```diff
@@ -33,6 +33,12 @@ const (
 	workQueueRetryLimit = 3
 )
 
+var (
+	invalidChars           = regexp.MustCompile(`[^a-z0-9.\-]`)
```

Comment:
```suggestion
	invalidChars           = regexp.MustCompile(`[^a-z0-9\.\-]`)
```

Reply:
@ivankatliarchuk - missing a back slash here⬆️

Reply:
Inside a character class ([...]) in a Go regular expression, the dot (.) does not need to be escaped because it loses its special meaning. So, \. and `.` are equivalent in this context. 

Shell I add it, or leave it as is?